main.go

```diff
@@ -85,16 +87,17 @@ func main() {
 		OutputPaths:      []string{"stdout"},
 		ErrorOutputPaths: []string{"stderr"},
 	}
-	logger, err := zapConfig.Build()
+	ZapLogger, err := zapConfig.Build()
 	if err != nil {
 		log.Fatal(err)
 	}
 
+	logger := zapr.NewLogger(ZapLogger)
```

Comment:
Now that we are using `log` instead of `logger` shouldn't this be `log` as well?

Reply:
But this means we use 2 different loggers. Can we just eliminate the use od the logger from "log"?

Reply:
This is only with main, using `log` to replace two statements like `fmt.Println() and exist (1)`with `log.Fatal` which is not supported by logger. if you think we can remove import of `log` and switch to fmt.Println and exist(1), i can do it. let me know if need to be changed

Reply:
why can we just use the logger we creating and use it. 
To replace the `fatal` just use the logger and then `panic` or `exit`

Reply:
done, replaced standard log with the logger and os.Exit(1)